README.md

```diff
@@ -175,14 +175,32 @@ kubectl delete --ignore-not-found customresourcedefinitions \
 #### Running *end-to-end* tests on local kind cluster:
 
 1. `kind create cluster --image=kindest/node:<latest>`. e.g `v1.23.0` version.
+
+> Note: In case you are running kind on MacOS using podman, it is recommended to create podman machine 4 CPUs and 8GiB memory.
+> Less resources might cause end to end tests fail because of lack of resources for cluster.
+>
+>  `podman machine init --cpus=4 --memory=8192 --rootful --now`
+>
+
 2. `kubectl cluster-info --context kind-kind`. kind version >= 0.6.x
 3. `make image` - build Prometheus Operator docker image locally.
+
+> Note: In case you are running kind using podman, the step 3 won't work for you. You will need to switch command in Makefile:
+>
+>  `sed -i 's/docker build/podman build/g' Makefile`
```

Comment:
this could also be "fixed" in the Makefile by adding a variable with a default value of `docker`:

```
DOCKER_CLI ?= docker
...
.hack-operator-image: Dockerfile operator
    $(DOCKER_CLI) build
```

Then you can run `DOCKER_CLI=podman make image`

Reply:
I did not. Thanks!